### PR TITLE
Resolve Issue #25

### PR DESCRIPTION
**Pull request by andnil5**
Resolves issue #25.
Creates an env.py while setting up the repo.

### CODE REVIEW

**Comment by omidhzr** [comment]
Looks great!

File: src/ci_utils.py [hunk 1]
```diff
@@ -27,14 +27,22 @@ def change_dir(dir):
     os.chdir(dir)
 
 
+def create_env_file():
+    """Creates an environment file with a TOKEN constant."""
+    f = open('src/env.py', 'w')
+    f.write('TOKEN = \'\'\n')
+    f.close()
```
Comment (omidhzr) [inline src/ci_utils.py:34]: We did not think about it, better to have it this way, well done Anders!

**Review by omidhzr — APPROVED** [review]

File: src/ci_utils.py (hunk 1 above)
Comment (jenniferlindberg) [reply src/ci_utils.py:34]: I agree with Omid, looks great!

**Review by jenniferlindberg — APPROVED** [review]
Looks great! Good that you also fixed some minor styling.

**Review by CalleFreme — APPROVED** [review]
Great catch!

**Pull request merged**